eras/mary/impl/testlib/Test/Cardano/Ledger/Mary/ImpTest.hs

```diff
@@ -52,3 +56,15 @@ instance
   , Signable (DSIGN c) (Hash (HASH c) EraIndependentTxBody)
   ) =>
   MaryEraImp (MaryEra c)
+
+mkTokenMintingTx :: MaryEraImp era => ScriptHash (EraCrypto era) -> ImpTestM era (Tx era)
+mkTokenMintingTx sh = do
+  name <- arbitrary
+  count <- choose (0, 10)
```

Comment:
Count can't be `0` in a MultiAsset, we go through great lengths in trying to preserve this invariant. We probably should hide the constructor from being exported in order to enforce this even further.
```suggestion
  count <- choose (1, 10)
```

Here is a function you can use to prevent 0 values from being constructed: https://github.com/IntersectMBO/cardano-ledger/blob/03c1bdff08fff2f9bcdeceb7e59113e4b79c9b1f/eras/mary/impl/src/Cardano/Ledger/Mary/Value.hs#L904-L907